README.md

```diff
@@ -20,3 +20,17 @@ the supported languages.
 ## `metadata.yaml`
 
 All `metadata.yaml` shall pass validation against `hack/metadata.schema.json` see [here](https://json-schema.org/) for more details
+
+## Release Cycle
+
+Managed Scripts has the following release cycle:
+
+### Staging:
+
+The staging environment is pinned to consume the main branch of the managed-scripts repository.
+
+### Production:
+
+Once every 3 weeks.
+
+For more information, please refer [Backplane Release Cycles](https://source.redhat.com/groups/public/sre/wiki/backplane_user_documentation)
```

Comment:
Same as the backplane-cli, I do not think we need to put the internal link in the public repo

Reply:
Updated